app/models/user.rb

```diff
@@ -2,11 +2,10 @@ class User < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
```

Comment:
Please style Sign Up`, and `Sign In` pages as per the design page. They have no style currenly.

User when clicks on `Sign up` button there is an error and app does not route the user to categories page. Kindly see the error below.

![sign_up_error](https://user-images.githubusercontent.com/35504149/204051291-ee8ffa76-cbac-43f4-a8a4-3fb4e262c6c0.png)